### ISSUE TITLE
Feature request: Wildcards for notification list

### ISSUE THREAD

**Issue by nbbsn** [AHAAAAAAA/PokemonGo-Map#1544]
I'd love a wildcard feature for the "Notify of Pokémon" filter so you'd get a notification anytime a Pokémon spawns that is not filtered out by the "Hide common Pokémon" filter. 
I think this would especially help players like myself who live in smaller towns that do not see a lot of pokemon spawns.   
Adding every Pokémon to the notification list manually is pretty tedious.

**Comment by Halvo317** [comment]
Yeah. Removing the -i and -o options was a huge mistake in my opinion.
